### ISSUE TITLE
NTR: Biliverdin high in urine (Lab: urine tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8863]
New term request Biliverdin high in urine (elthat_130418043956):

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000913
- label: Elevated urine biliverdin level
- parents: [Abnormal urine metabolite level (HP:0033354)](https://hpo.jax.org/app/browse/term/HP:0033354)
- pmid: 
- def: The amount of biliverdin in the urine, normalized for urine concentration, is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8863
- synonyms:
